src/effects/effectchainslot.cpp

```diff
@@ -244,6 +244,17 @@ EffectChainPointer EffectChainSlot::getEffectChain() const {
     return m_pEffectChain;
 }
 
+EffectChainPointer EffectChainSlot::getAndEnsureEffectChain(
+        EffectsManager* pEffectsManager) {
+    if (!m_pEffectChain) {
+        auto pEffectChain = EffectChainPointer(
+                new EffectChain(pEffectsManager, QString()));
+        pEffectChain->setName(QObject::tr("Empty Chain"));
```

Comment:
I was not aware of this trick, nice to know ;) But also a little weird, because the String now appears in the context of QObject. "Empty chain" might not be associated with _effects_ by the string itself.